nginx/nginx.conf

```
@@ -69,7 +69,19 @@ server {
 		proxy_redirect     off;
 	}
 
-	location /sockjs-node {
+	location /admin {
+		proxy_http_version 1.1;
+		proxy_set_header   Upgrade $http_upgrade;
+		proxy_set_header   Connection 'upgrade';
+		proxy_set_header   X-Forwarded-For $remote_addr;
+		proxy_set_header   X-Real-IP  $remote_addr;
+		proxy_set_header   Host $host;
+		
+		proxy_pass         http://admin:8080;
```

```suggestion
		proxy_pass         http://admin:5000;
```

This is just a suggestion, but shouldn't we port all our applications like 3000, 4000, 5000 ?